code/modules/mob/living/basic/bots/medbot/medbot.dm

```diff
@@ -335,12 +349,12 @@
 		log_combat(src, patient, "tended the wounds of", "internal tools")
 		if(patient.get_current_damage_of_type(damage_type_healer) <= heal_threshold)
 			done_healing = TRUE
+
 	patient.visible_message(span_notice("[src] tends the wounds of [patient]!"), "[span_infoplain(span_green("[src] tends your wounds!"))]")
-	//Go into idle only when we're done
+	update_bot_mode(new_mode = BOT_IDLE)
```

Comment:
The reason this was moved into done_healing is so they wouldn't switch animations when not done. Without it being in there they'll constantly go from syringe out to syringe in.

Reply:
it introduced a bug where sometimes if u move out of the way of an AI controlled  medbot while still injured right after the healing action, it would get stuck in the healing animation but im going to fix it some otherway